apps/crawler/crawler/s3/crawler/s3_crawler.py

```diff
@@ -28,7 +38,18 @@ def __init__(
                 )
 
     def crawl(self) -> None:
-        self._find_and_download_new_objects()
+        try:
+            self._s3_client = self._get_s3_client()
+            self._find_and_download_new_objects()
+            return
+        except NoCredentialsError:
+            if self._anon:
+                raise
+            # Retry anonymously...
```

Comment:
logging.Warning("Automatically retrying in anonymous mode")

Reply:
This file doesn't use `logging` anywhere else.  Is it OK to `print`?